source/Coop.Core/Client/Services/Settlements/Handlers/ClientSettlementHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Common.Logging;
+using Common.Messaging;
+using Common.Network;
+using Coop.Core.Client.Services.Settlements.Messages;
+using Coop.Core.Server.Services.Settlements.Messages;
+using GameInterface.Services.Settlements.Messages;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Coop.Core.Client.Services.Settlements.Handlers
+{
+    /// <summary>
+    /// Client handler for settlements
+    /// </summary>
+    public class ClientSettlementHandler : IHandler
```

Comment:
This name is too generic, please update to include `SettlementOwnership` and also update comments